tools/vagrant/README.md

```diff
@@ -168,24 +167,6 @@ vagrant ssh
 wskadmin user create <subject>
 ```
 
-This command will create a new *subject* with the authorization key shown on the console once you run `wskadmin`. This key is required when making API calls to OpenWhisk, or when using the command line interface (CLI). The namespace is the same as the `<subject>` name used to create the key.
```

Comment:
that just adds one subject with one namespace.
the text here allows you to add namespaces to the same subject.
it should be restored (although i think all these text should be moved really why is it in the vagrant readme; separate pr).